iina/PrefCodecViewController.swift

```diff
@@ -99,4 +103,21 @@ class PrefCodecViewController: PreferenceViewController, PreferenceWindowEmbedda
     let hwdec: Preference.HardwareDecoderOption = Preference.enum(for: .hardwareDecoder)
     hwdecDescriptionTextField.stringValue = hwdec.localizedDescription
   }
+
+  @IBAction func updateEnableToneMapping(_ sender: NSButton) {
```

Comment:
Can you do this in xib by bindings?

Reply:
Yes, I'm changing this to binding enabling of the two controls to the checkbox in the XIB.